extensions/standard-processors/tests/unit/ExecuteProcessTests.cpp

```diff
@@ -170,6 +170,9 @@ TEST_CASE_METHOD(ExecuteProcessTestsFixture, "ExecuteProcess buffer long outputs
   CHECK(success_flow_files[0]->getAttribute("command") == command);
   CHECK(success_flow_files[0]->getAttribute("command.arguments") == arguments);
 }
-
+#else
+TEST_CASE("ExecuteProcessTests") {
+  SUCCEED("To avoid empty test executable");
+}
```

Comment:
It may be better to remove the `#ifdef` and remove the test file in the `CMakeLists.txt` with `list(REMOVE_ITEM ...)` if we are building on Windows.

Reply:
Good idea, I've fixed it in https://github.com/apache/nifi-minifi-cpp/pull/1615/commits/44c7d79b3d130f483271f92ef7b8b224533e4137